nodejs/package-lock.json

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
Do we want to check in the package lock? Or would it be better for this to be excluded and autogenerated?

Reply:
`package-lock.json` should be committed and is suggested by `npm` official doc.

https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json

> This file is intended to be committed into source repositories, and serves various purposes:

`npm ci` will use the version in `package-lock.json` and produces a deterministic build.

*This may change if we release platform-specific builds as different platforms need different packages versions.*